stdlib/public/core/BridgeStorage.swift

```diff
@@ -65,7 +74,11 @@ internal struct _BridgeStorage<NativeClass: AnyObject> {
   @inlinable
   @inline(__always)
   internal init(taggedPayload: UInt) {
+    #if !$Embedded
     rawValue = _bridgeObject(taggingPayload: taggedPayload)
+    #else
+    rawValue = Builtin.reinterpretCast(taggedPayload)
```

Comment:
Hm, I believe directly casting between reference types and trivial types is undefined behavior. If we need to keep this type enabled, then this reinterpret cast probably ought to be replaced with:

```suggestion
    let p = UnsafeRawPointer(bitPattern: taggedPayload)
    self.rawValue = Unmanaged.fromOpaque(p).takeUnretainedValue()
```

(The `takeUnretainedValue()` will emit a (depening on the value, probably, but not necessarily) noop call to swift_retain.)

However, the notion of initializing a regular strong reference from an arbitrary bit pattern is bogus; we should not do it, ever. The only users of this entry point are `_modify` accessors and the like, where it is used to generate dummy instances. Instead of using a null pointer for that purpose, we need to start using the immortal empty singleton object we already have!